backend/metadata_writer/requirements.in

```diff
@@ -1,2 +1,3 @@
 kubernetes>=8.0.0,<11.0.0
 ml-metadata==1.5.0
+lru-dict>=1.1.7
```

Comment:
I've just adopted this dependency to make life easier for myself, happy to drop this and write a little helper util to handle this if you prefer not to take on the dependency?